### ISSUE TITLE
Hide ListGroupHeaders when there are little files

### ISSUE THREAD

**Issue by TheScientist** [nextcloud/windows-universal#42]
For me the ListGroupHeader for file (A-Z) is confusing when there are less than 10 files in a folder.
I'd prefer to hide them in that case.

**Comment by SunboX** [comment]
Good point. The number should be configurable and there should also be a setting to disable group headers completely.
